cupy/linalg/eigenvalue.py

```diff
@@ -121,7 +120,10 @@ def eigh(a, UPLO='L'):
 
     .. seealso:: :func:`numpy.linalg.eigh`
     """
-    return _syevd(a, UPLO, True)
+    if a.ndim == 3:
+        return cusolver._syevj_batched(a, UPLO, True)
```

Comment:
Thank you for your PR.
Could you change `eigh` to support arbitrary `a.ndim > 2` cases like as follows?:
```suggestion
    if a.ndim > 2:
        *batch_shape, m, n = a.shape
        w, v = cusolver._syevj_batched(a.reshape(-1, m, n), UPLO, True)
        return w.reshape(*batch_shape, m), v.reshape(a.shape)
```